### DISCUSSION TITLE
Compilation fails

### DISCUSSION THREAD

**Discussion by Liudeke** [sofa-framework/sofa#2626] [Build / Config / Environment] [answered]
![image](https://user-images.githubusercontent.com/30310819/150457277-d55596ba-f294-4d83-be89-9b96086aa1a8.png)
I don't know why this is happening，I tried a lot of things, but nothing worked

**Comment by fredroy** [comment, marked as answer]
It seems the linker did not use libglew (so it is missing the symbols for it)
In your cmake configuration, what does it say for the variable `GLEW_SHARED_LIBRARY_RELEASE` ?

**Reply by Liudeke** [reply]
thanks,I've solved the problem

**Reply by fredroy** [reply]
Good to know 👍
